packages/@cdktf/hcl2json/lib/syntax-tree.ts

```diff
@@ -192,6 +197,11 @@ export type BinaryOpExpression = CommonExpressionAst & {
   meta: BinaryOpExpressionMeta;
 };
 
+export type ObjectExpression = CommonExpressionAst & {
+  type: "object";
+  meta: ObjectExpressionMeta; // Doesn't have any special meta attributes
```

Comment:
What does this comment mean? It seems like it has `items` – remove the comment?

Reply:
forgot to remove the comment